### PR TITLE
Update norman and KEv2 operators

### PR DESCRIPTION
**Pull request by cmurphy**
These updates allow KEv2 cluster specs to set `norman:"pointer"` on
fields to explicitly designate them as pointers in the generated client
code. This applies to string pointers, maps, and slices.

Depends on:
- https://github.com/rancher/norman/pull/400
- https://github.com/rancher/gke-operator/pull/49
- https://github.com/rancher/eks-operator/pull/38

https://github.com/rancher/rancher/issues/32440
https://github.com/rancher/rancher/issues/32609
https://github.com/rancher/rancher/issues/33153

### CODE REVIEW

**Comment by cmurphy** [comment]
Added commit to address https://github.com/rancher/rancher/issues/32553

**Review by thedadams — DISMISSED** [review]

**Review by cbron — APPROVED** [review]

**Review by rmweir — APPROVED** [review]
LGTM

**Comment by cmurphy** [comment]
Had to fix go.sum

**Review by rmweir — DISMISSED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -88,12 +88,12 @@ require (
 	github.com/prometheus/common v0.10.0
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
-	github.com/rancher/eks-operator v1.0.6
-	github.com/rancher/gke-operator v1.0.2-rc1
+	github.com/rancher/eks-operator v1.1.1-rc3
```
Comment (cbron) [inline go.mod:91]: Is EKS operator on the same version as gke ? I thought it was still on 1.0.x for 2.5

Comment (cmurphy) [reply go.mod:91]: Will have to backport in order to keep the 1.0.x version https://github.com/rancher/eks-operator/pull/43

**Review by thedadams — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (cbron) [reply go.mod:91]: Yes we decided that EKS operator should stay at 1.0.x for 2.5.x and move to 2.0.0 for 2.6

Comment (cbron) [reply go.mod:91]: I see that we wen't with 1.1.1 for 2.6 at first, that will change to 2.0.0 and will be tagged of master. So any fixes that need to go into 2.5 should be backported to 1.0.x

**Comment by cmurphy** [comment]
Changed eks-operator to 1.0.8-rc3

File: go.mod [hunk 2]
```diff
@@ -88,12 +88,12 @@ require (
 	github.com/prometheus/common v0.10.0
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
-	github.com/rancher/eks-operator v1.0.6
-	github.com/rancher/gke-operator v1.0.2-rc1
+	github.com/rancher/eks-operator v1.0.8-rc3
```
Comment (thedadams) [inline go.mod:91]: Can we use v1.0.8?

**Review by thedadams — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (cmurphy) [reply go.mod]: Changed to v1.0.9

**Review by thedadams — DISMISSED** [review]

**Review by thedadams — APPROVED** [review]

**Review by mjura — APPROVED** [review]
LGTM

**Review by rmweir — APPROVED** [review]
LGTM

**Pull request merged**